src/MerchantFrontend/src/app/views/merchant-vouchers/merchant-vouchers.component.html

```diff
@@ -0,0 +1,30 @@
+<h1>Gutscheine</h1>
+
+<!-- Datagrid Table -->
+<clr-datagrid (clrDgRefresh)="refresh()" [clrDgLoading]="loading">
+  <clr-dg-placeholder>Keine Gutscheine gefunden.</clr-dg-placeholder>
+
+  <!-- Columns -->
+  <clr-dg-column [clrDgField]="'name'">
```

Comment:
I noticed that the clrDgField is for the buildin search. I think we should remove it because a little symbol is shown in the grid.